_maps/RandomRuins/SpaceRuins/oldteleporter.dmm

```diff
@@ -63,7 +63,6 @@
 /area/ruin/space/oldteleporter)
 "n" = (
 /obj/machinery/power/apc{
-	dir = 0;
```

Comment:
I haven't tested but there's a chance these APC and intercom changes will mess with pixel offsets. Notably APCs will be pixel offset in their specified dir and changing that from `0` to `SOUTH` will change where the APCs are positioned. Fine if all these APCs are south anyways.

Reply:
From what I can tell, they're all fine.